qalculator/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.1.3)
+
+project(qalculator)
+
+file(GLOB_RECURSE SRC src/*)
+
+find_package(Qt5 5.5.0 REQUIRED COMPONENTS Widgets)
```

Comment:
Add `find_package(qalculate REQUIRED)` to fail if qalculate is not installed.

Reply:
qalculate does not provide a cmake file on most distros. I added proper pkgconf loading now